### DISCUSSION TITLE
Using high-volume events such as InputVolumeMeters

### DISCUSSION THREAD

**Discussion by mccombey** [Palakis/obs-websocket#1152] [Q&A]
I've been using obs-websocket.js v5 for several months and everything seems to work fine but I **CANNOT** work out how to use the high-volume events, such as InputVolumeMeters. The documentation doesn't appear to show how these work either. Does anyone have an example of using InputVolumeMeters they could share, so I can measure the output volume from OBS on a regular basis?

Any help would be very much appreciated.

**Comment by Maxou44** [comment]
Same issue on my side

**Comment by bASEstr** [comment]
Necroing a little bit, but it seems like this is the first google result when you search "how to use InputVolumeMeters obs" and so I figured actually explaining how would help others searching for the same topic.

What you need to do is during [identify](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#identify-opcode-1) you need to activate the high volume event that you want (in this case, InputVolumeMeters event) by byte masking the eventSubscriptions field, and only then you will receive the event packet.
[This](https://github.com/obsproject/obs-websocket/issues/152#issuecomment-1722471280) is a code example code from a year or so ago in a related question.

**Comment by faithoflifedev** [comment]
Thanks @bASEstr, your info was helpful in allowing me to subscribe to the events.

The resulting record I have is:

{"inputs":[
{"inputLevelsMul":[
[0.000047171954065561295,0.00013339084398467094,0.0011836696648970246],[0.000049328955356031656,0.0001347525103483349,0.0011957527603954077]
],
"inputName":"Media Source","inputUuid":"52b8a81e-4035-4aa2-9c85-34fc20e1388b"}
]}

Can anyone tell me how I should interpret the values for `inputLevelsMul`?
